src/dashboards/components/dashboard_index/DashboardCard.tsx

```diff
@@ -231,11 +231,13 @@ class DashboardCard extends PureComponent<Props> {
   private handleDeleteDashboard = async () => {
     const {id, name, onDeleteDashboard} = this.props
```

Comment:
since these are getting destructured anyway, to save some space should we destructure `isPinned` and `sendNotification` here too?

Reply:
For consistency that might be a good idea, but I'd rather get the bugfix in rather than go through another round of CI (which itself is an indication that our CI takes too long to run).